### ISSUE TITLE
Converting pre-trained models from Caffe to PyTorch

### ISSUE THREAD

**Issue by abrosua** [sniklaus/pytorch-liteflownet#19]
First of all, thanks for your LiteFlowNet reimplementation in PyTorch. I wonder how do you convert the pre-trained model to PyTorch?

Since I'm trying to convert this [model](https://github.com/shengzesnail/PIV-LiteFlowNet-en) from Caffe to PyTorch also, could you please explain the procedure, or maybe refer to a certain method that you used? That would be very helpful.

Regards

**Comment by sniklaus** [comment]
Thank you for your kind words and for being interested in this repository! I first examined the Caffe model defined in the prototxt. In doing so, I always find [Netscope](https://dgschwend.github.io/netscope/quickstart.html) to be quite handy. All that was left to do then was to load the pre-trained Caffe weights in the PyTorch model. To do so, I used [loadcaffe](https://github.com/szagoruyko/loadcaffe) to load the Caffe weights and to store them as Torch weights. I was then finally able to use [torchfile](https://github.com/bshillingford/python-torchfile) to load the Torch weights in PyTorch. There probably is a more direct approach though.
